{"name":"dragTestcase","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [true] but found [false]","trace":"java.lang.AssertionError: expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:110)\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\n\tat org.testng.Assert.assertTrue(Assert.java:56)\n\tat org.testng.Assert.assertTrue(Assert.java:66)\n\tat com.codecraft.flutter.DragNDropTest.DragNDropTest.dragTestcase(DragNDropTest.java:20)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\n\tat org.testng.TestRunner.run(TestRunner.java:602)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\n\tat org.testng.TestNG.run(TestNG.java:1067)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","steps":[],"attachments":[{"name":"Failure in method {0}","source":"5fe941d5-916b-4a89-9c0a-0936b15c68f7-attachment","type":"image/png"},{"name":"dragTestcase failed and screenshot taken!","source":"100bb49d-fe22-426d-b5de-588d9363e20d-attachment","type":"text/plain"}],"parameters":[{"name":"environment","value":"IphoneLocal"}],"start":1683444284546,"stop":1683444301462,"uuid":"1fd42f1c-825c-4263-a2ab-50abca4bb870","historyId":"756b7b70347cc0a79f8624fa6e3e4ba8","fullName":"com.codecraft.flutter.DragNDropTest.DragNDropTest.dragTestcase","labels":[{"name":"package","value":"com.codecraft.flutter.DragNDropTest.DragNDropTest"},{"name":"testClass","value":"com.codecraft.flutter.DragNDropTest.DragNDropTest"},{"name":"testMethod","value":"dragTestcase"},{"name":"parentSuite","value":"Flutter Mobile Automation "},{"name":"suite","value":"FLUTTER-4: Regression"},{"name":"subSuite","value":"com.codecraft.flutter.DragNDropTest.DragNDropTest"},{"name":"host","value":"Ostans-MacBook-Pro-3.local"},{"name":"thread","value":"88135@Ostans-MacBook-Pro-3.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}